var/spack/repos/builtin/packages/py-sphinxcontrib-moderncmakedomain/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySphinxcontribModerncmakedomain(PythonPackage):
+    """Sphinx Domain for Modern CMake."""
+
+    homepage = "https://github.com/scikit-build/moderncmakedomain"
+    pypi = "sphinxcontrib_moderncmakedomain/sphinxcontrib_moderncmakedomain-3.25.0.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("3.25.0", sha256="4138e4d3f60e5c4b3982caa10033693bfc1009cdd851766754d5990d9d1e992a")
```

Comment:
There is a newer version (3.27.0) available as well in case you want to add it.

Reply:
Thank you, I add the two last newer version: 3.27.0 and 3.26.4